mlfinlab/data_structures/data_structures.py

```diff
@@ -30,11 +31,11 @@ def _update_counters(cache, flag):
     # Check flag
     if flag and cache:
         # Update variables based on cache
-        cum_ticks = int(cache[-1][6])
-        cum_dollar_value = np.float(cache[-1][5])
-        cum_volume = cache[-1][4]
-        low_price = np.float(cache[-1][2])
-        high_price = np.float(cache[-1][3])
+        cum_ticks = int(cache[-1].cum_ticks)
```

Comment:
`cache[-1]` is not intuituve. Consider to write it into separate variable and leave short inline comment